### PR TITLE
Add single unit test for React component

### PR DESCRIPTION
**Pull request by grahse**

### CODE REVIEW

File: client/src/components/Pledge.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        backed: 0,
+        aboutMe: 'About Me'
+      },
+      options: []
+    }
+    this.fetchOwner = this.fetchOwner.bind(this);
+    this.fetchOptions = this.fetchOptions.bind(this);
+  }
+
+  componentDidMount() {
```
Comment (jamesstockman14) [inline client/src/components/Pledge.jsx:23]: I think we are supposed to move these invocations into the constructor and not keep them in componentDidMount. Leslie suggested that during one of the Q&A's.

File: tests/Pledge.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+describe("Pledge component", () => {
+  test("renders", () => {
+    const div = document.createElement('div');
+    div.id = "pledge";
+    const wrapper = shallow(<Pledge />);
+
+    expect(wrapper.exists()).toBe(true);
+    expect(wrapper.find(ProjectOwner)).toHaveLength(1);
+    // expect(wrapper.find('PledgeOption')).toHaveLength(4);
```
Comment (jamesstockman14) [inline tests/Pledge.test.js:18]: Is this comment still necessary?

**Review by jamesstockman14 — COMMENTED** [review]
Nice work Grace :) No major changes just a few suggestions.

File: client/src/components/Pledge.jsx (hunk 1 above)
Comment (grahse) [reply client/src/components/Pledge.jsx:23]: Good tip, I'll have to check that out!

File: tests/Pledge.test.js (hunk 2 above)
Comment (grahse) [reply tests/Pledge.test.js:18]: Nope, still working on tests so the next PR will definitely update that, haha

**Pull request merged**